### ISSUE TITLE
Allow for multiple invocations of compile! rather than a single one

### ISSUE THREAD

**Issue by patham9** [trueagi-io/metta-morph#20]
Currently only a single compile! call is supported, though can include multiple functions within the compiled code.

It should however be possible to just build and load additional .so files, the challenge though is that each references the Chicken Scheme .so files, which it seems can only be referenced once in a process. There is likely a way around this though!

**Comment by Necr0x0Der** [comment]
This is a good feature/issue for analyzing within #15

**Comment by patham9** [comment]
I agree, #15 should be investigated first as #20 might turn out to be time-consuming.
